src/amchart/CommonXY.js

```diff
@@ -57,6 +97,10 @@
         this._chart.theme = "none";
         this._chart.type = "xy";
         this._chart.pathToImages = "//cdn.rawgit.com/cdnjs/cdnjs/master/ajax/libs/amcharts/3.13.0/images/";
```

Comment:
itll work but its going to throw errors saying it cant find the images ... going to remove